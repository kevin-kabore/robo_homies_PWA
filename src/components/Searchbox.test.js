import { shallow } from 'enzyme';
import React from 'react';

import SeacrchBox from './SearchBox';

let wrapper;
beforeEach(() => {
	wrapper = shallow(<SeacrchBox />);
})
describe('<SeacrchBox />', () => {
	it('should render the component', () => {
		expect(wrapper).toMatchSnapshot();
	})
})