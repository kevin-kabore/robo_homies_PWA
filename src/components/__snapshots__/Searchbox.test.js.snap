// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SeacrchBox /> should render the component 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SearchBox />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <input
        aria-label="Search Robots"
        className="pa3 ba b--green bg black"
        onChange={undefined}
        placeholder="search robot homies"
        type="search"
/>,
      "className": "pa2",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "aria-label": "Search Robots",
        "className": "pa3 ba b--green bg black",
        "onChange": undefined,
        "placeholder": "search robot homies",
        "type": "search",
      },
      "ref": null,
      "rendered": null,
      "type": "input",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <input
          aria-label="Search Robots"
          className="pa3 ba b--green bg black"
          onChange={undefined}
          placeholder="search robot homies"
          type="search"
/>,
        "className": "pa2",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "aria-label": "Search Robots",
          "className": "pa3 ba b--green bg black",
          "onChange": undefined,
          "placeholder": "search robot homies",
          "type": "search",
        },
        "ref": null,
        "rendered": null,
        "type": "input",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
